zadanie3/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>A Web Page</title>
+  </head>
+
+  <body>
+    <header role="banner">
```

Comment:
Wzorcowy `<header>`! :+1: Jeśli mam się czepiać detali, to zrzuciłabym tagi `<a>` do osobnych linijek.